bindata/network/ovn-kubernetes/common/002-rbac.yaml

```diff
@@ -13,16 +13,22 @@ metadata:
 rules:
 - apiGroups: [""]
   resources:
-  - endpoints
-  - namespaces
   - pods
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
+- apiGroups: [""]
+  resources:
+  - namespaces
+  - endpoints
   - services
   - configmaps
```

Comment:
We don't need access to all configmaps across all namespaces. Can you add a separate Role (not ClusterRole) for the openshift-ovn-kubernetes namespace?